shell/scl_source

```diff
@@ -7,9 +7,9 @@ Don't use this script outside of SCL scriptlets!
 Options:
     -h, --help    display this help and exit"
 
-if [ $# -eq 0 -o $1 = "-h" -o $1 = "--help" ]; then
+if [ $# -eq 0 ] || [ -o "$1" = "-h" ] || [ -o "$1" = "--help" ]; then
     echo "$_scl_source_help"
-    return 0
+    return 1
```

Comment:
This is actually API change.  The `scl_source` command can fail `set -e` scripts now if someone forgot to add arguments, e.g. by `scl_source "${@}"` (if there's no argument) => before it was just no-op.

Reply:
Good point. Thanks

Reply:
This was changed based on this comment https://github.com/sclorg/scl-utils/pull/10#discussion_r101695544

Reply:
Yes, I was wrong before, I think.